liblepton/scheme/conf/upgrade.scm

```diff
@@ -0,0 +1,233 @@
+;; Lepton EDA
+;; liblepton - Lepton's library - Scheme API
+;; Copyright (C) 2019 dmn <graahnul.grom@gmail.com>
+;; License: GPLv2+. See the COPYING file
+;;
+
+( define-module ( conf upgrade )
+
+  #:use-module  ( ice-9 format )
+  #:use-module  ( geda  config )
+  #:use-module  ( conf  keys   )
+
+  #:export      ( convert )
```

Comment:
I would prefer more descriptive names like, say, `upgrade-config` and avoid using of common names for procedures, since they could clash with imported third party code.